test/functorch/test_vmap.py

```diff
@@ -4085,7 +4084,7 @@ def f(e_):
         xfail('linalg.norm'),  # can accept vector inputs
         xfail('linalg.norm', 'subgradients_at_zero'),  # can accept vector inputs
         skip('linalg.multi_dot'),  # accepts list of tensor inputs, has its own special test
-        xfail('linalg.vander'),
+        xfail('linalg.vander'), # cannot accept dimensions equal to or less than one
```

Comment:
Should this be failing?

Reply:
Maybe I misinterpreted the way linalg_vander works, but reading the code, it looks like it checks for whether the size of the input Tensor is greater than 1

Reply:
The comment should probably be `# can accept vector inputs`, as in the ones above it.